### PR TITLE
RHPAM-3058: Improve error message on Runtime UI for the missing kieserver configuration

### PR DESCRIPTION
**Pull request by jesuino**
**JIRA**:RHPAM-3058

[link](https://issues.redhat.com/browse/RHPAM-3058)

This is an improvement of how error messages are displayed on Business Central and Dashbuilder Runtime. It providers a card with error information and allow users to check the error details. See how it looks like:

** During Dashboards Authoring:

image

** On Dashbuilder Runtime

image

With this we hope that it will be easier for users to understand what is wrong with their dashboards and possibly share error details in community forums or with support team.

### CODE REVIEW

File: dashbuilder/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/DisplayerErrorWidget.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  -->
+<div class="card-pf card-pf-view card-pf-view-xs displayer-error"
+    data-field="displayerErrorRoot">
+    <div class="card-pf-body">
+        <div class="card-pf-heading">
+            <span
+                class="pficon pficon-warning-triangle-o displayer-error-title-icon"></span><span
+                class="card-pf-title"
+                data-i18n-key="displayerErrorTitle">Unexpected
+                Error</span>
```
Comment (caponetto) [inline dashbuilder/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/DisplayerErrorWidget.html:24]:
```suggestion
                data-i18n-key="displayerErrorTitle">
                </span>
```

File: dashbuilder/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/DisplayerErrorWidget.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        </div>
+        <p data-field="errorBody" class="displayer-error-body"></p>
+        <div>
+            <a data-field="chevronDown" hidden><em
+                class="glyphicon glyphicon-chevron-down"></em></a> <a
+                data-field="chevronRight"><em
+                class="glyphicon glyphicon-chevron-right"></em></a> <span
+                class="chevron-title" data-i18n-key="errorDetails">Details</span>
```
Comment (caponetto) [inline dashbuilder/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/DisplayerErrorWidget.html:32]:
```suggestion
                class="chevron-title" data-i18n-key="errorDetails"></span>
```

**Comment by jesuino** [comment]
@caponetto removed the texts

**Comment by jesuino** [comment]
jenkins execute fdb

**Review by barboras7 — APPROVED** [review]
Looks good, thanks @jesuino!

**Comment by jesuino** [comment]
Failed tests are unrelated:


org.kie.karaf.itest.KieSpringjBPMPersistenceKarafIntegrationTest.testJbpmRuntimeManagerWithPersistence
org.kie.karaf.itest.KieSpringjBPMPersistenceKarafIntegrationTest.testJbpmRuntimeManager
org.kie.server.springboot.samples.KieServerTest.testProcessStartAndWorkOnUserTask
org.kie.server.springboot.samples.KieServerMigrationTest.testProcessStartAndAbort

**Pull request merged**